tips/TIP-0045/tip-0045.md

```diff
@@ -0,0 +1,1101 @@
+  </tr>
+  <tr>
+    <td>Input Type</td>
+    <td>uint8</td>
+    <td>Set to <strong>value 2</strong> to denote a <i>Block Issuance Credit Input</i>.</td>
+  </tr>
+  <tr>
+    <td>Account ID</td>
+    <td>ByteArray[32]</td>
+    <td>The ID of the Account for which this input provides the BIC.</td>
+  </tr>
+</table>
+
+##### Reward Input
+
+A _Reward Input_ indicates which transaction `Input` is claiming Mana rewards. It can reference an _Account Output_ with
+a _Staking Feature_ or a _Delegation Output_. The input is resolved by calculating the total amount of rewards the
+respective output can claim which is provided as context for transaction execution. The amount of rewards that can be
+claimed is added to the total sum of Mana on the input side of the transaction. Multiple such inputs can be present in a
+single transaction to claim rewards for different outputs; however, no two _Reward Inputs_ can reference the same index.
+
+It is serialized as follows:
+
+<details>
+  <summary>Reward Input</summary>
+  <blockquote>A Reward Input indicates which transaction Input is claiming Mana rewards.</blockquote>
+</details>
+<table>
+  <tr>
+    <td>
+      <b>Name</b>
+    </td>
+    <td>
+      <b>Type</b>
+    </td>
+    <td>
+      <b>Description</b>
+    </td>
+  </tr>
+  <tr>
+    <td>Input Type</td>
+    <td>uint8</td>
+    <td>Set to <strong>value 3</strong> to denote a <i>Reward Input</i>.</td>
+  </tr>
+  <tr>
+    <td>Index</td>
+    <td>uint16</td>
+    <td>The index of the transaction input for which to claim rewards.</td>
+  </tr>
+</table>
+
+#### Outputs
+
+The `Outputs` field holds the outputs that are created by the <i>Transaction Payload</i>. There are different output
+types, but they must all have an `Amount` field denoting the number of IOTA coins to deposit.
+
+#### Allotments
+
+The `Allotments` field contains the list of all Mana allotments, the `Account ID`, and corresponding values that
+converts Mana provided by the inputs in the form of stored Mana in inputs or potential Mana derived from the inputs'
+IOTA coins. Mana listed in this field will be added upon commitment of the slot in which the transaction was issued, in
+form of _Block Issuance Credits_ to the account's BIC value.
+
+Note that _Block Issuance Credits_ are used to pay for the block issuance, they are burned on the slot commitment of the
+issuance slot. The good practice would be to always allot enough Mana to cover for the block issuance.
+
+#### Payload
+
+The _Transaction_ itself can contain another payload as described in general in [TIP-46](../TIP-0046/tip-0046.md). The
+[semantic validity](#semantic-validation) of the encapsulating _Transaction Payload_ is unaffected by the possibly
+contained payload.
+
+### Unlocks
+
+The `Unlocks` field holds the unlocks which unlock the inputs in a _Transaction_.
+
+#### Signature Unlock
+
+The <i>Signature Unlock</i> holds a signature signing the BLAKE2b-256 hash of the <i>Transaction</i> (including the
```

Comment:
Can you elaborate what is being signed exactly? Is it just the hash of the transaction serialization bytes or is it one of the TransactionId components?

Reply:
Clarified in the TIP. It's the serialized `Transaction` field.